### PR TITLE
add jenkins iib parameter support

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: ci_jobs_trigger/libs/jenkins/utils/general.py [hunk 1]
```diff
@@ -1,23 +1,54 @@
...
         return False, None
 
+    try:
+        last_build_number = api.get_job_info(job)["lastBuild"]["number"]
+    except TypeError:
+        last_build_number = 0
+
+    api.build_job(name=job, parameters=set_job_params(api=api, job=job, operator_iib=operator_iib))
+
+    return wait_for_job_in_jenkins(api=api, job=job, last_build_number=last_build_number, logger=logger)
```
Comment (myakove) [inline ci_jobs_trigger/libs/jenkins/utils/general.py:23]:
```suggestion
    return wait_for_job_started_in_jenkins(api=api, job=job, last_build_number=last_build_number, logger=logger)
```

File: ci_jobs_trigger/libs/jenkins/utils/general.py [hunk 2]
```diff
@@ -1,23 +1,54 @@
...
-        return build.exists(), build
-    except Exception:
-        return False, None
+    for _property in api.get_job_info(name=job)["property"]:
+        for param in _property.get("parameterDefinitions", []):
+            if operator_iib and param["name"] == install_from_iib_job_param_str:
+                job_params[install_from_iib_job_param_str] = True
+                continue
+
+            job_params[param["defaultParameterValue"]["name"]] = param["defaultParameterValue"]["value"]
```
Comment (myakove) [inline ci_jobs_trigger/libs/jenkins/utils/general.py:36]: `param["defaultParameterValue"]["name"]` can be `param["name"]` ?

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by myakove — DISMISSED** [review]

**Review by myakove — APPROVED** [review]

**Comment by redhat-qe-bot2** [comment]
New container for quay.io/redhat_msi/ci-jobs-trigger:latest published

**Pull request merged**